config/settings.yml

```diff
@@ -27,6 +27,11 @@ github_notification_monitor:
   repo_names: []
 merge_target_titler:
   enabled_repos: []
+path_based_labeler:
+  enabled_repos: []
+  rules: []
+    # - pattern:
+    #   label:
```

Comment:
Then we just have to update Repo#enabled_for? to:

```ruby
  def enabled_for?(checker)
    enabled_repos = Settings[checker].enabled_repos || []
    enabled_repos = Array(enabled_repos) if enabled_repos.kind_of?(String)
    enabled_repos.include?(name)
  end
```

There might be a more elegant way to do this, but it's eluding me.

These tests should all pass (which I think all of the above does)

- enabled_repos: nil
- enabled_repos: single_string
- enabled_repos: ["array", "of, "string"]
- enabled_repos: {"hash" => "of, "key" => "values"}